src/main/java/exception/InvalidFormatException.java

```diff
@@ -0,0 +1,7 @@
+package exception;
+
+public class InvalidFormatException extends RuntimeException {

```

Comment:
I like how you have two `Exception` classes with semantic names that clearly describe the error they are representing. Just one feedback - as @alushingg mentioned, it would be good to add header comments to all public classes. For example, I'd love to know what are the possible situations that will throw an `InvalidFormatException`, since I may be unsure what is the format you're referring to exactly.